package.json

```diff
@@ -30,7 +30,12 @@
     "build:umd:min": "cross-env BABEL_ENV=es NODE_ENV=production rollup -c -i src/index.js -o dist/redux-loop.min.js",
     "build": "yarn build:commonjs && yarn build:es && yarn build:umd && yarn build:umd:min",
     "prebuild": "yarn run clear",
-    "prepublishOnly": "yarn lint && yarn test && yarn build"
+    "prepublishOnly": "yarn lint && yarn test && yarn build",
+    "docs:clean": "rimraf _book",
+    "docs:prepare": "gitbook install",
+    "docs:build": "npm run docs:prepare && gitbook build -g redux-loo/redux-loop && cp logo/logo.png _book/gitbook/images/logo.png && cp logo/favicon.ico _book/gitbook/images",
+    "docs:watch": "npm run docs:prepare && gitbook serve",
+    "docs:publish": "npm run docs:clean && npm run docs:build && cp CNAME _book && cd _book && git init && git commit --allow-empty -m 'update book' && git checkout -b gh-pages && touch .nojekyll && git add . && git commit -am 'update book' && git push git@github.com:redux-loo/redux-loop gh-pages --force"
```

Comment:
can you add a file to the root of the repo called CNAME that contains `redux-loop.js.org`? This task is looking for that file. After this is working I'll get that url from https://dns.js.org/